### PR TITLE
Update Mixcloud plugin

### PR DESCRIPTION
**Pull request by LaneSh4d0w**
Mixcloud deployed an update to their UI, making this plugin presence obsolete to its state. This is a try into making it work once again.

### CODE REVIEW

**Comment by callumok2004** [comment]
Please attach a screenshot showing the newly updated code in action, otherwise we wont know it works.

**Comment by LaneSh4d0w** [comment]
I will try to do so.

**Comment by LaneSh4d0w** [comment]
I'll close it from the time being. I'll open it back once I get the screenshots, to prevent other plugins being done.

**Comment by LaneSh4d0w** [comment]
When user watches a live stream on Mixcloud Live : 

Mixcloud #1

When user listens to an uploaded show : 

Mixcloud #2

Needed to delete anything related to timestamps, to reduce the program load.

File: websites/M/Mixcloud/presence.ts [hunk 1]
```diff
@@ -13,12 +13,13 @@ presence.on("UpdateData", async () => {
...
-    const paused =
-      document.querySelector(
+    const paused, on_air;
+
+    paused = document.querySelector(
         ".PlayButton__PlayerControl-rvh8d9-1"
       ) === null;
 
-    const on_air = document.querySelector(".LiveStreamPage__LiveStreamPageHeader-hwzri8-1");
+    on_air = document.querySelector(".LiveStreamPage__LiveStreamPageHeader-hwzri8-1");
```
Comment (Bas950) [inline websites/M/Mixcloud/presence.ts]: That is not how you merge consts...

```
const var1 = obj,
var2 = obj2;
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Mixcloud/presence.ts (hunk 1 above)
Comment (LaneSh4d0w) [reply websites/M/Mixcloud/presence.ts]: My bad, had understood it like this.

File: websites/M/Mixcloud/presence.ts [hunk 2]
```diff
@@ -1,54 +1,65 @@
...
-        elapsedSec = parseInt(elapsed[0]) * 60 + parseInt(elapsed[1]);
-      }
+const presence = new Presence({
+  clientId: "610102236374368267"
+});
+const strings = presence.getStrings({
+  play: "presence.playback.playing",
+  pause: "presence.playback.paused",
+  live: "presence.activity.live"
+});
```
Comment (Bas950) [inline websites/M/Mixcloud/presence.ts:8]: merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
bump version in metadata

I thought you did already :thonk:

**Comment by LaneSh4d0w** [comment]
Wasn't even thinking of that haha, was simply reworking the presence. Will do.

File: websites/M/Mixcloud/dist/metadata.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "author": {
-    "name": "Eai",
+    "name": "Eai / LaneSh4d0w",
     "id": "132779412771962880"
```
Comment (Bas950) [inline websites/M/Mixcloud/dist/metadata.json:4]: thats not how you add yourself as a contributor see: https://docs.premid.app/en/dev/presence/metadata

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Mixcloud/dist/metadata.json (hunk 3 above)
Comment (LaneSh4d0w) [reply websites/M/Mixcloud/dist/metadata.json:4]: Whoops.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Contributors should be in an array

**Review by Bas950 — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Just seen it, made the changes.

**Review by Alanexei — APPROVED** [review]

**Pull request merged**